shared/whats-new/versions.tsx

```diff
@@ -34,9 +36,27 @@ export const VersionTitle = ({title}: {title: string}) => (
   </Kb.Box2>
 )
 
-export const Current = ({seen}: VersionProps) => {
+export const Current = ({onSwitchTab, seen}: VersionProps) => {
```

Comment:
this is fine but you don't need to do all this plumbing. you can just use Contianer.useDispatch and dispatch the action here